src/dotnet/commands/dotnet-run/LaunchSettings/ILaunchSettingsProvider.cs

```diff
@@ -1,13 +1,13 @@
-﻿using Microsoft.DotNet.Cli.Utils;
-using Newtonsoft.Json.Linq;
+﻿using System.Text.Json;
+using Microsoft.DotNet.Cli.Utils;
 
 namespace Microsoft.DotNet.Tools.Run.LaunchSettings
 {
     internal interface ILaunchSettingsProvider
     {
         string CommandName { get; }
 
-        LaunchSettingsApplyResult TryApplySettings(JObject document, JObject model, ref ICommand command);
+        LaunchSettingsApplyResult TryApplySettings(JsonElement document, JsonElement model, ref ICommand command);
```

Comment:
It seems a little odd to me to have a parameter named `document` whose type is `JsonElement` (vs `JsonDocument`)

Reply:
This has been removed